pkg/image/apis/image/dockerpre012/dockertypes.go

```diff
@@ -53,3 +55,80 @@ type DockerConfig struct {
 	// This field is not supported in pre012 and will always be empty.
 	Labels map[string]string `json:"Labels,omitempty"`
 }
+
+// ImagePre012 serves the same purpose as the Image type except that it is for
+// earlier versions of the Docker API (pre-012 to be specific)
+// Exists only for legacy conversion, copy of type from fsouza/go-dockerclient
+type ImagePre012 struct {
```

Comment:
This is required to pin our serialization. Since we expose this, this API is ours and we need to own our destiny.